plenum/test/txn_author_agreement/helper.py

```diff
@@ -1,10 +1,24 @@
+from typing import NamedTuple, Dict
+
 from indy.ledger import build_txn_author_agreement_request
 
-from plenum.common.constants import CONFIG_LEDGER_ID
+from plenum.common.constants import (
+    CONFIG_LEDGER_ID,
+    TXN_AUTHOR_AGREEMENT_TEXT, TXN_AUTHOR_AGREEMENT_VERSION,
+    TXN_PAYLOAD, TXN_METADATA, TXN_METADATA_SEQ_NO, TXN_METADATA_TIME
+)
 from plenum.common.util import randomString
 from plenum.server.config_req_handler import ConfigReqHandler
 
 
+TaaData = NamedTuple("TaaData", [
+    ("version", str),
+    ("text", str),
+    ("seqNo", int),
+    ("txnTime", int)
```

Comment:
Here is another suggestion to use snake_case

Reply:
fixed